seatunnel-connectors-v2/connector-elasticsearch/src/main/java/org/apache/seatunnel/connectors/seatunnel/elasticsearch/client/EsRestClient.java

```diff
@@ -376,8 +376,7 @@ public Map<String, String> getFieldTypeMapping(String index, List<String> source
                     mapping = getFieldTypeMappingFromProperties(properties, source);
                 } else {
                     for (Iterator<JsonNode> iter = mappingsProperty.iterator(); iter.hasNext(); ) {
-                        JsonNode typeNode = iter.next();
-                        JsonNode properties = typeNode.get("properties");
+                        JsonNode properties = iter.next();
```

Comment:
It maybe has some problems on es5 cc @iture123

Reply:
@TyrantLucifer Indeed, the mapping of es has changed, this problem has been fixed and the test has passed es5, es6, es7+
https://github.com/apache/incubator-seatunnel/blob/ac50b31ca3a534649cb90b467a7654f6e52f7180/seatunnel-connectors-v2/connector-elasticsearch/src/main/java/org/apache/seatunnel/connectors/seatunnel/elasticsearch/client/EsRestClient.java#L378-L387